x/upgrade/client/cli/query.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/x/upgrade"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd creates a query sub-command for the upgrade module using cmdName as the name of the sub-command.
+func GetQueryCmd(cmdName string, storeName string, cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   cmdName,
```

Comment:
maybe we call this `list` or `show`? the command would be `gaiacli q upgrade show`

Reply:
So in Regen Ledger it's `xrncli query upgrade-plan` so no separate sub-command, but happy to add one if that makes sense. `show` would work

Reply:
Well here on the SDK we use the module client interface to export and automatically register CLI functionality. See `gov` for an example https://github.com/cosmos/cosmos-sdk/blob/develop/x/gov/client/module_client.go#L17

Reply:
I was going to do that initially but I'm not exporting any tx command for this module because it should be handled by gov. I guess clients could filter nil values here: https://github.com/cosmos/cosmos-sdk/blob/develop/cmd/gaia/cmd/gaiacli/main.go#L150